sql/database.go

```diff
@@ -172,12 +140,41 @@ func WithQueryCacheSizes(sizes map[QueryCacheKind]int) Opt {
 	}
 }
 
+// WithForceMigrations forces database to run all the migrations instead
+// of using a schema snapshot in case of a fresh database.
+func WithForceMigrations(force bool) Opt {
+	return func(c *conf) {
+		c.forceMigrations = true
+	}
+}
+
+// WithSchema specifies database schema script.
+func WithDatabaseSchema(schema *Schema) Opt {
+	return func(c *conf) {
+		c.schema = schema
+	}
+}
+
+// WithIgnoreTableRx specifies regular expression for table names to ignore during schema
+// drift detection.
+func WithIgnoreTableRx(rx string) Opt {
+	return func(c *conf) {
+		c.ignoreTableRx = rx
+	}
+}
+
+func withForceFresh(fresh bool) Opt {
+	return func(c *conf) {
+		c.forceFresh = fresh
+	}
+}
```

Comment:
As `forceFresh` is disabled by default, how about:

```suggestion
func withForceFresh() Opt {
	return func(c *conf) {
		c.forceFresh = true
	}
}
```

Reply:
ok will fix, thanks

Reply:
Fixed